Dockerfile.QA

```diff
@@ -99,7 +99,6 @@ RUN mkdir -p qa/common && \
     cp -r docs/examples/model_repository/simple qa/L0_cuda_shared_memory/models/. && \
     mkdir qa/L0_client_java/models && \
     cp -r docs/examples/model_repository/simple qa/L0_client_java/models && \
-    mkdir qa/L0_grpc/models && \
     cp -r docs/examples/model_repository/simple qa/L0_grpc/models && \
```

Comment:
Not sure - but this seems like it might still be needed?

Reply:
Good catch! Fixed.